@xen-orchestra/xapi/network.mjs

```diff
@@ -0,0 +1,51 @@
+import { asyncEach } from '@vates/async-each'
+import { createLogger } from '@xen-orchestra/log'
+
+const { warn } = createLogger('xo:xapi:network')
+
+export default class Network {
+  async setMtu(network, mtu) {
+    await network.set_MTU(mtu)
+
+    // The MTU will not be updated for VIFs of paused VM, but we can't unplug/replug VIF on a paused VM
+    const pluggedVifsRefs = network.$VIFs
```

Comment:
Code in this lib should not rely on cached records, explicitly fetch records/fields you need.